src/systems/air_speed/AirSpeed.cc

```diff
@@ -101,6 +110,15 @@ AirSpeed::AirSpeed() :
 //////////////////////////////////////////////////
 AirSpeed::~AirSpeed() = default;
 
+//////////////////////////////////////////////////
+void AirSpeed::Configure(const Entity &_entity,
+    const std::shared_ptr<const sdf::Element> &_sdf,
+    EntityComponentManager &_ecm,
+    EventManager &)
```

Comment:
CI picked up a couple more warnings about unused parameters:

```suggestion
void AirSpeed::Configure(const Entity &,
    const std::shared_ptr<const sdf::Element> &,
    EntityComponentManager &_ecm,
    EventManager &)
```